core/app/models/concerns/spree/user_methods.rb

```diff
@@ -23,6 +23,15 @@ module UserMethods
       has_many :store_credit_events, through: :store_credits
       money_methods :total_available_store_credit
 
+
+      # Specifies an amount in days which incomplete orders will
+      # not be returned by #last_incomplete_spree_order based on when
+      # it was last updated. By default it will not be set, and
+      # continue to return all incomplete orders.
+      # @api public
+      class_attribute :completable_order_cutoff
+      self.completable_order_cutoff = nil
```

Comment:
I think this would be a good one for `Spree::AppConfiguration` since the concept isn't tiedt directly to the implementation of the class and it would make sense to have even if all this code was refactored.